README.md

```diff
@@ -16,9 +16,20 @@ Run the container
 docker run -it -p 11211:11211 --name memcached modularitycontainers/memcached
 ```
 
-If you would like to increase a CACHE_SIZE use environment variable -e CACHE_SIZE:
+If you would like to debug memcached, use container option =e MEMCACHED_DEBUG_MODE:
 ```bash
-docker run -it -e CACHE_SIZE=128 -p 11211:11211 --name memcached modularitycontainers/memcached
+docker run -it -p 11211:11211
```

Comment:
I just tried this and am getting zero output: so how is this option helpful in debugging?

```
$ docker run --rm -ti -e MEMCACHED_DEBUG_MODE modularitycontainers/memcached
```

Reply:
You have missed -p 11211:11211. But nevertheless, I have to find out proper way how to debug memcached.